server/db/users.js

```diff
@@ -1,29 +1,34 @@
-const connection = require('./')
+const environment = process.env.NODE_ENV || 'development'
```

Comment:
Please use what you removed in line 1 as the next few lines are from the index file it imports.